css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,70 @@
+.container {
+  padding: 10px;
+}
+
+.row {
+  text-align: center;
+  padding: 5px;
+  font-family:
+    "DM Sans",
+    sans-serif;
```

Comment:
The only styling on the `row` class should be `display: flex`. You have that applied to the `display` class rule below this. Please revise this to follow proper layout classes protocol.